Windows/ppsspp.rc

```diff
@@ -68,6 +68,7 @@ BEGIN
     "W",            ID_EMULATION_STOP,      VIRTKEY, CONTROL, NOINVERT
     "B",            ID_EMULATION_RESET,     VIRTKEY, CONTROL, NOINVERT
     "T",            ID_EMULATION_CHEATS,    VIRTKEY, CONTROL, NOINVERT
+	"C",            ID_EMULATION_CHAT,      VIRTKEY, CONTROL, NOINVERT

```

Comment:
Use spaces in this file.  You might install the `editorconfig` plugin for your editor of choice.

Also below.

-[Unknown]